templates/dovecot/conf.d/10-master.conf.j2

```diff
@@ -72,12 +72,22 @@ service imap {
 
   # Max. number of IMAP processes (connections)
   process_limit = 1024
+
+{% if mailserver_dove_version.stdout|version_compare('2.2.19', '>=') %}
```

Comment:
The task setting this var should have ```check_mode: False``` set

Reply:
Fixed in 2a48b2d.